homeassistant/components/google_assistant/helpers.py

```diff
@@ -86,19 +86,19 @@ def _get_registry_entries(
 class AbstractConfig(ABC):
     """Hold the configuration for Google Assistant."""
 
+    _store: GoogleConfigStore
```

Comment:
```suggestion
    _store: GoogleConfigStore | None = None
```

Reply:
With the current typing it's not allowed to use a Google `AbstractConfig` instance until `AbstractConfig.async_initialize` has been awaited.
If changed like this we instead need to add None-checks everywhere `self._store` is accessed and somehow handle that there's no store.